src/lib/questions-utils.property.spec.ts

```
@@ -0,0 +1,185 @@
+import { describe, it, expect, vi } from "vitest";
```

ESLint

[error] 'vi' is defined but never used.

(@typescript-eslint/no-unused-vars)